lib/feature_flags.js

```diff
@@ -8,7 +8,6 @@
 // unreleased flags gating an active feature
 exports.prerelease = {
   express5: false,
-  new_promise_tracking: false,
```

Comment:
you actually need to move it to the released array below

Reply:
Done with 8e095bf.